src/Cache/Engine/FileEngine.php

```diff
@@ -41,7 +41,7 @@ class FileEngine extends CacheEngine
      *
      * @var \SplFileObject
      */
-    protected SplFileObject $_File;
+    protected SplFileObject $File;
```

Comment:
Renaming such props from `$File` to `$file` is also something we could do separately.